origamiUROP/polygons.py

```diff
@@ -56,17 +54,25 @@ def perp_vector(self) -> np.ndarray:
         """Perpendicular vector which lies in the xy plane"""
         return np.cross(self.unit_vector, np.array([0, 0, 1]))
 
-    def strand(self, **kwargs) -> Strand:
-        no_of_nucleotides_in_edge = int(self.length)
-        ssDNA = generateHelix(
+    def strand(self, sequence : str = None, **kwargs) -> Strand:
```

Comment:
`generate_helix` returns a `List` containing 1 or 2 `Strand`'s, can we still annotate it as `Strand`? It makes sense to me, but just double checking

Reply:
actually we should do the following:

```python
from typing import List

def function() -> List[Strand]:
     return list_of_strands
```